recipes-support/xorg-xrdp/xorgxrdp_%.bbappend

```
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

> xorgxrdp_0.9.%.bbappend
> xrdp_0.9.%.bbappend

Also, can you make the bbappend not version specific? i.e.,

> xorgxrdp_%.bbappend
> xrdp_%.bbappend

If the version changes, we would still want these changes applied. And if they don't apply cleanly, I think it's preferable that it fail the build rather than the warning we get on orphaned bbappend which may not get noticed.

---

Done